.env.EXAMPLE

```diff
@@ -1,3 +1,7 @@
 DB_NAME='user_db'
 DB_USER='user'
-DB_PASSWORD='password'
\ No newline at end of file
+DB_PASSWORD='password'
+
+PORT=3001
+TWILIO_API_KEY=35ba8ac0c3743bccbf70bc1b187a802c
```

Comment:
Lets take these out and add them to our own env files. We don't want these to be public